### DISCUSSION TITLE
Not working?

### DISCUSSION THREAD

**Discussion by at-ng** [ehudhala/logseq-plugin-daily-todo#2] [General]
So I installed this yesterday and at the end of the day my latest journal looked like this:

- plan
	- plan 1
	- plan 2
- done
	- DONE task 1
	- DONE task 2
	- TODO task 3
	- TODO task 4
	- TODO task 5
	- LATER task 6
	- LATER task 7
	- LATER task 8
	- LATER task 9
	- TODO task 10
	- LATER task 11
	    task 11 details

But when the new journal was created today it was completely empty as usual. Maybe I've misunderstood something but I thought it was supposed to look something like this:

- done
	- TODO task 3
	- TODO task 4
	- TODO task 5
	- LATER task 6
	- LATER task 7
	- LATER task 8
	- LATER task 9
	- TODO task 10
	- LATER task 11
	    task 11 details

**Edit:** Today it did something but instead of moving everything that it was supposed to everything was moved **including the unrelated "plan" section**. The tasks marked as done were not moved to the new journal but there where not left in the original one either so **they were removed completely.**

**Comment by joaomotap** [comment]
I've just opened Logseq to discover that all TODOs and DOINGs that were pending from the previous day are gone, and nowhere to be found...

**Comment by OddSquirrel** [comment]
For me, it does move the TODOs to the next day, but deletes all DONEs from the previous one, so I have no way to find out what and when something was actually done.

Too bad this plugin was abandoned, it would have done a great job if functional. Obviously, something must have changed inside LogSeq in the meantime. Does anybody know if there's a way to contact LogSeq so it gets removed from the plugin store? Doesn't really make sense to offer it while it's broken.

**Reply by at-ng** [reply]
https://logseq.com/ [Contact Us](mailto:hi@logseq.com)
https://discuss.logseq.com/

https://github.com/logseq/logseq/issues
https://github.com/logseq/logseq/discussions

**Reply by xymsyh** [reply]
EN: Yes this plugin is broken and it's killing me as well
ZH: 是的这个插件坏了，这也让我很苦恼

**Comment by ehudhala** [comment]
Hi everyone!
The plugin had a bug with TODO items nested under DONE items that is fixed in the latest release (0.0.10).
Also development is back and a lot of work has been done on stability of the plugin with a new end to end test suite.
If you can reproduce a bug with expected structure of both pages pre-migrate and post migrate we can make sure they're included in the end to end tests and work as expected.
